### PR TITLE
DROOLS-5803 Fix kie-camel dep not in MavenCentral xercesImpl 2.12.0.SP02

### PR DESCRIPTION
**Pull request by tarilabs**
showing the reproducer setup in simpler terms.

```
$ mvn clean test -Dtest=DroolsJaxbHelperXSDInKJARTest

[INFO]
[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ kie-camel
---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running
org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest
15:07:05.897 [main] WARN  o.a.m.i.embedder.MavenSettings - Environment
variable M2_HOME is not set
Nov 26, 2020 3:07:07 PM com.sun.xml.bind.v2.util.XmlFactory
createSchemaFactory
SEVERE: null
org.xml.sax.SAXNotRecognizedException:
http://javax.xml.XMLConstants/feature/secure-processing
	at javax.xml.validation.SchemaFactory.setFeature(Unknown Source)
	at com.sun.xml.bind.v2.util.XmlFactory.createSchemaFactory(XmlFactory.java:110)
	at com.sun.tools.xjc.reader.xmlschema.parser.SchemaConstraintChecker.check(SchemaConstraintChecker.java:86)
	at com.sun.tools.xjc.ModelLoader.loadXMLSchema(ModelLoader.java:342)
	at com.sun.tools.xjc.ModelLoader.load(ModelLoader.java:162)
	at com.sun.tools.xjc.ModelLoader.load(ModelLoader.java:117)
	at org.drools.compiler.runtime.pipeline.impl.DroolsJaxbHelperProviderImpl.addXsdModel(DroolsJaxbHelperProviderImpl.java:138)
	at org.drools.compiler.runtime.pipeline.impl.DroolsJaxbHelperProviderImpl.addPackageFromXSD(DroolsJaxbHelperProviderImpl.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.drools.dynamic.DynamicComponentsSupplier.addPackageFromXSD(DynamicComponentsSupplier.java:94)
	at org.drools.reflective.ComponentsFactory.addPackageFromXSD(ComponentsFactory.java:53)
	at org.drools.compiler.builder.impl.KnowledgeBuilderImpl.addPackageFromXSD(KnowledgeBuilderImpl.java:797)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl$ResourceBuilder.lambda$static$1(CompositeKnowledgeBuilderImpl.java:288)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl$ResourceBuilder$$Lambda$49/000000001C952860.build(Unknown
Source)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.buildResourceType(CompositeKnowledgeBuilderImpl.java:139)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.buildResources(CompositeKnowledgeBuilderImpl.java:128)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.build(CompositeKnowledgeBuilderImpl.java:103)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.build(CompositeKnowledgeBuilderImpl.java:97)
	at org.drools.compiler.kie.builder.impl.AbstractKieProject.buildKnowledgePackages(AbstractKieProject.java:268)
	at org.drools.compiler.kie.builder.impl.AbstractKieProject.buildKnowledgePackages(AbstractKieProject.java:216)
	at org.drools.compiler.kie.builder.impl.AbstractKieProject.verify(AbstractKieProject.java:80)
	at org.drools.compiler.kie.builder.impl.KieBuilderImpl.buildKieProject(KieBuilderImpl.java:279)
	at org.drools.compiler.kie.builder.impl.KieBuilderImpl.buildAll(KieBuilderImpl.java:247)
	at org.drools.compiler.kie.builder.impl.KieBuilderImpl.buildAll(KieBuilderImpl.java:194)
	at org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest.testXsdModelInKJAR(DroolsJaxbHelperXSDInKJARTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

15:07:07.188 [main] INFO  o.k.c.e.c.c.DroolsJaxbHelperXSDInKJARTest -
class com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl
15:07:07.189 [main] INFO  o.k.c.e.c.c.DroolsJaxbHelperXSDInKJARTest -
class com.sun.org.apache.xerces.internal.jaxp.validation.XMLSchemaFactory
Nov 26, 2020 3:07:07 PM com.sun.xml.bind.v2.util.XmlFactory
createSchemaFactory
SEVERE: null
org.xml.sax.SAXNotRecognizedException:
http://javax.xml.XMLConstants/feature/secure-processing
	at javax.xml.validation.SchemaFactory.setFeature(Unknown Source)
	at com.sun.xml.bind.v2.util.XmlFactory.createSchemaFactory(XmlFactory.java:110)
	at org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest.testInternalsDryRun(DroolsJaxbHelperXSDInKJARTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

[ (edited) ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed:
1.838 s <<< FAILURE! - in
org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest
[ (edited) ERROR] testXsdModelInKJAR(org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest)
Time elapsed: 1.777 s  <<< ERROR!
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.drools.dynamic.DynamicComponentsSupplier.addPackageFromXSD(DynamicComponentsSupplier.java:102)
	at org.drools.reflective.ComponentsFactory.addPackageFromXSD(ComponentsFactory.java:53)
	at org.drools.compiler.builder.impl.KnowledgeBuilderImpl.addPackageFromXSD(KnowledgeBuilderImpl.java:797)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl$ResourceBuilder.lambda$static$1(CompositeKnowledgeBuilderImpl.java:288)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl$ResourceBuilder$$Lambda$49/000000001C952860.build(Unknown
Source)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.buildResourceType(CompositeKnowledgeBuilderImpl.java:139)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.buildResources(CompositeKnowledgeBuilderImpl.java:128)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.build(CompositeKnowledgeBuilderImpl.java:103)
	at org.drools.compiler.builder.impl.CompositeKnowledgeBuilderImpl.build(CompositeKnowledgeBuilderImpl.java:97)
	at org.drools.compiler.kie.builder.impl.AbstractKieProject.buildKnowledgePackages(AbstractKieProject.java:268)
	at org.drools.compiler.kie.builder.impl.AbstractKieProject.buildKnowledgePackages(AbstractKieProject.java:216)
	at org.drools.compiler.kie.builder.impl.AbstractKieProject.verify(AbstractKieProject.java:80)
	at org.drools.compiler.kie.builder.impl.KieBuilderImpl.buildKieProject(KieBuilderImpl.java:279)
	at org.drools.compiler.kie.builder.impl.KieBuilderImpl.buildAll(KieBuilderImpl.java:247)
	at org.drools.compiler.kie.builder.impl.KieBuilderImpl.buildAll(KieBuilderImpl.java:194)
	at org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest.testXsdModelInKJAR(DroolsJaxbHelperXSDInKJARTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.drools.dynamic.DynamicComponentsSupplier.addPackageFromXSD(DynamicComponentsSupplier.java:94)
	... 43 more
Caused by: java.lang.IllegalStateException:
org.xml.sax.SAXNotRecognizedException:
http://javax.xml.XMLConstants/feature/secure-processing
	at com.sun.xml.bind.v2.util.XmlFactory.createSchemaFactory(XmlFactory.java:114)
	at com.sun.tools.xjc.reader.xmlschema.parser.SchemaConstraintChecker.check(SchemaConstraintChecker.java:86)
	at com.sun.tools.xjc.ModelLoader.loadXMLSchema(ModelLoader.java:342)
	at com.sun.tools.xjc.ModelLoader.load(ModelLoader.java:162)
	at com.sun.tools.xjc.ModelLoader.load(ModelLoader.java:117)
	at org.drools.compiler.runtime.pipeline.impl.DroolsJaxbHelperProviderImpl.addXsdModel(DroolsJaxbHelperProviderImpl.java:138)
	at org.drools.compiler.runtime.pipeline.impl.DroolsJaxbHelperProviderImpl.addPackageFromXSD(DroolsJaxbHelperProviderImpl.java:110)
	... 48 more
Caused by: org.xml.sax.SAXNotRecognizedException:
http://javax.xml.XMLConstants/feature/secure-processing
	at javax.xml.validation.SchemaFactory.setFeature(Unknown Source)
	at com.sun.xml.bind.v2.util.XmlFactory.createSchemaFactory(XmlFactory.java:110)
	... 54 more

[ (edited) ERROR] testInternalsDryRun(org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest)
Time elapsed: 0.009 s  <<< ERROR!
java.lang.IllegalStateException: org.xml.sax.SAXNotRecognizedException:
http://javax.xml.XMLConstants/feature/secure-processing
	at com.sun.xml.bind.v2.util.XmlFactory.createSchemaFactory(XmlFactory.java:114)
	at org.kie.camel.embedded.camel.component.DroolsJaxbHelperXSDInKJARTest.testInternalsDryRun(DroolsJaxbHelperXSDInKJARTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: org.xml.sax.SAXNotRecognizedException:
http://javax.xml.XMLConstants/feature/secure-processing
	at javax.xml.validation.SchemaFactory.setFeature(Unknown Source)
	at com.sun.xml.bind.v2.util.XmlFactory.createSchemaFactory(XmlFactory.java:110)
	... 29 more
```

**JIRA**: https://issues.redhat.com/browse/DROOLS-5803

**referenced Pull Requests**: none

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by tarilabs** [comment]
Jenkins retest this

**Comment by tarilabs** [comment]
As of 3fedf0a it would fail for IBM JDK

**Comment by tarilabs** [comment]
whlie 2371372 is a proper fix that would allow to drop the dependency of xercesImpl also for IBM JDK

**Comment by tarilabs** [comment]
I am not getting why
```
org.kie.maven.plugin.AdditionalPropertiesIntegrationTest.testAdditionalPropertiesCorrectlySet[3.3.9]
org.kie.maven.plugin.AdditionalPropertiesIntegrationTest.testAdditionalPropertiesCorrectlySet[3.5.0]
org.kie.maven.plugin.BuildMojoIntegrationTest.testCleanInstallWithAllSupportedResourceTypes[3.3.9]
org.kie.maven.plugin.BuildMojoIntegrationTest.testCleanInstallWithAllSupportedResourceTypes[3.5.0]
org.kie.maven.plugin.InjectReactiveIntegrationTest.testBasicBytecodeInjection[3.3.9]
org.kie.maven.plugin.InjectReactiveIntegrationTest.testBasicBytecodeInjectionSelected[3.3.9]
org.kie.maven.plugin.InjectReactiveIntegrationTest.testBasicBytecodeInjection[3.5.0]
org.kie.maven.plugin.InjectReactiveIntegrationTest.testBasicBytecodeInjectionSelected[3.5.0]
org.kie.maven.plugin.KJarWithDMNIntegrationTest.testComplexDMNModelWithExecutableModel[3.3.9]
org.kie.maven.plugin.KJarWithDMNIntegrationTest.testCleanInstall[3.3.9]
org.kie.maven.plugin.KJarWithDMNIntegrationTest.testComplexDMNModel[3.3.9]
org.kie.maven.plugin.KJarWithDMNIntegrationTest.testComplexDMNModelWithExecutableModel[3.5.0]
org.kie.maven.plugin.KJarWithDMNIntegrationTest.testCleanInstall[3.5.0]
org.kie.maven.plugin.KJarWithDMNIntegrationTest.testComplexDMNModel[3.5.0]
org.kie.maven.plugin.AlphaNetworkCompilerTest.testAlphaNetworkCompiler[3.3.9]
org.kie.maven.plugin.AlphaNetworkCompilerTest.testAlphaNetworkCompiler[3.5.0]
org.kie.maven.plugin.BuildPMMLTest.testUseBuildKjarWithPMML[3.3.9]
org.kie.maven.plugin.BuildPMMLTest.testContentKjarWithPMML[3.3.9]
org.kie.maven.plugin.BuildPMMLTest.testUseBuildKjarWithPMML[3.5.0]
org.kie.maven.plugin.BuildPMMLTest.testContentKjarWithPMML[3.5.0]
org.kie.maven.plugin.DeclaredTypesTest.testDeclaredTypeWithJavaField[3.3.9]
org.kie.maven.plugin.DeclaredTypesTest.testDeclaredTypeWithJavaFieldExecutableModel[3.3.9]
org.kie.maven.plugin.DeclaredTypesTest.testDeclaredTypeWithJavaField[3.5.0]
org.kie.maven.plugin.DeclaredTypesTest.testDeclaredTypeWithJavaFieldExecutableModel[3.5.0]
org.kie.maven.plugin.ExecModelParameterTest.testNoMavenWithDroolsModelCompilerOnClassPathDoNotRunExecModel[3.3.9]
org.kie.maven.plugin.ExecModelParameterTest.testDefaultMavenWithDroolsModelCompilerOnClassPathRunsExecModel[3.3.9]
org.kie.maven.plugin.ExecModelParameterTest.testYesMavenWithDroolsModelCompilerOnClassPathRunsExecModel[3.3.9]
org.kie.maven.plugin.ExecModelParameterTest.testDefaultMavenWithoutDroolsModelCompilerOnClassPathDoNotRunExecModel[3.3.9]
org.kie.maven.plugin.ExecModelParameterTest.testYesMavenWithoutDroolsModelCompilerOnClassPathDoNotRunExecModel[3.3.9]
org.kie.maven.plugin.ExecModelParameterTest.testNoMavenWithoutDroolsModelCompilerOnClassPathDoNotRunExecModel[3.3.9]
org.kie.maven.plugin.ExecModelParameterTest.testNoMavenWithDroolsModelCompilerOnClassPathDoNotRunExecModel[3.5.0]
org.kie.maven.plugin.ExecModelParameterTest.testDefaultMavenWithDroolsModelCompilerOnClassPathRunsExecModel[3.5.0]
org.kie.maven.plugin.ExecModelParameterTest.testYesMavenWithDroolsModelCompilerOnClassPathRunsExecModel[3.5.0]
org.kie.maven.plugin.ExecModelParameterTest.testDefaultMavenWithoutDroolsModelCompilerOnClassPathDoNotRunExecModel[3.5.0]
org.kie.maven.plugin.ExecModelParameterTest.testYesMavenWithoutDroolsModelCompilerOnClassPathDoNotRunExecModel[3.5.0]
org.kie.maven.plugin.ExecModelParameterTest.testNoMavenWithoutDroolsModelCompilerOnClassPathDoNotRunExecModel[3.5.0]
org.kie.maven.plugin.MultiModuleTest.testMultiModule[3.3.9]
org.kie.maven.plugin.MultiModuleTest.testMultiModuleWithExecutableModel[3.3.9]
org.kie.maven.plugin.MultiModuleTest.testMultiModule[3.5.0]
org.kie.maven.plugin.MultiModuleTest.testMultiModuleWithExecutableModel[3.5.0]
org.kie.maven.takari.plugin.AdditionalPropertiesIntegrationTakariTest.testAdditionalPropertiesCorrectlySet[3.3.9]
org.kie.maven.takari.plugin.AdditionalPropertiesIntegrationTakariTest.testAdditionalPropertiesCorrectlySet[3.5.0]
org.kie.maven.takari.plugin.BuildMojoIntegrationTakariTest.testCleanInstallWithAllSupportedResourceTypes[3.3.9]
org.kie.maven.takari.plugin.BuildMojoIntegrationTakariTest.testCleanInstallWithAllSupportedResourceTypes[3.5.0]
org.kie.maven.takari.plugin.InjectReactiveIntegrationTakariTest.testBasicBytecodeInjection[3.3.9]
org.kie.maven.takari.plugin.InjectReactiveIntegrationTakariTest.testBasicBytecodeInjectionSelected[3.3.9]
org.kie.maven.takari.plugin.InjectReactiveIntegrationTakariTest.testBasicBytecodeInjection[3.5.0]
org.kie.maven.takari.plugin.InjectReactiveIntegrationTakariTest.testBasicBytecodeInjectionSelected[3.5.0]
```

are allegedly failing, the seems to be invoking the Camel test ? 😮

**Comment by tarilabs** [comment]
I am suspecting is a security / code injection / failure to escape the environment properties,
like `ghprbPullLongDescription`
which is the content of this PR page and the commit.

To give it a shot, by rewording the content of the commit to not contain any stacktrace/commands.

**Comment by tarilabs** [comment]
```
19:49:40  [INFO] ------------------------------------------------------------------------
19:49:40  [INFO] BUILD FAILURE
19:49:40  [INFO] ------------------------------------------------------------------------
19:49:40  [INFO] Total time:  6.057 s (Wall Clock)
19:49:40  [INFO] Finished at: 2020-11-26T13:47:46-05:00
19:49:40  [INFO] ------------------------------------------------------------------------
19:49:40  [ERROR] Failed to execute goal on project jbpm-recommendation-pmml-logistic-regression: Could not resolve dependencies for project org.kie:jbpm-recommendation-pmml-logistic-regression:jar:7.47.0-SNAPSHOT: Could not transfer artifact com.sun.xml.fastinfoset:FastInfoset:jar:1.2.15 from/to mirror-central (https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/groups/kie-all/): /home/jenkins/.m2/repository/com/sun/xml/fastinfoset/FastInfoset/1.2.15/FastInfoset-1.2.15.jar.part (No such file or directory) -> [Help 1]
19:49:40  org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project jbpm-recommendation-pmml-logistic-regression: Could not resolve dependencies for project org.kie:jbpm-recommendation-pmml-logistic-regression:jar:7.47.0-SNAPSHOT: Could not transfer artifact com.sun.xml.fastinfoset:FastInfoset:jar:1.2.15 from/to mirror-central (https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/groups/kie-all/): /home/jenkins/.m2/repository/com/sun/xml/fastinfoset/FastInfoset/1.2.15/FastInfoset-1.2.15.jar.part (No such file or directory)
```

Jenkins retest this

**Comment by tarilabs** [comment]
As all the maven plugin testing are failing with something like:

```
[...]
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at io.takari.maven.testing.executor.MavenExecutionResult.assertErrorFreeLog(MavenExecutionResult.java:44)
	at org.kie.maven.plugin.KieMavenPluginBaseIntegrationTest.getMavenExecutionResult(KieMavenPluginBaseIntegrationTest.java:90)
	at org.kie.maven.plugin.KieMavenPluginBaseIntegrationTest.lambda$buildKJarProject$0(KieMavenPluginBaseIntegrationTest.java:80)
	at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1660)
	at org.kie.maven.plugin.KieMavenPluginBaseIntegrationTest.buildKJarProject(KieMavenPluginBaseIntegrationTest.java:80)
	at org.kie.maven.plugin.MultiModuleTest.testMultiModuleExec(MultiModuleTest.java:58)
	at org.kie.maven.plugin.MultiModuleTest.testMultiModule(MultiModuleTest.java:51)
```

Which I believe is due to:

https://github.com/kiegroup/droolsjbpm-integration/blob/b3efee193304d1610eeb961dd432a4486b57b87a/kie-plugins-testing/src/test/java/org/kie/maven/plugin/KieMavenPluginBaseIntegrationTest.java#L90

Which in turn is checked by Takari as:

https://github.com/takari/takari-plugin-testing-project/blob/dd441b849a2e6d1932f02dcaaf0c4238a960ec96/takari-plugin-testing/src/main/java/io/takari/maven/testing/executor/MavenExecutionResult.java#L40

I believe this was indeed due to LOG of maven being pasted in commits/PR page.

Editing previous comment to give this a shot

**Comment by tarilabs** [comment]
Jenkins retest this

**Review by jiripetrlik — APPROVED** [review]
LGTM. Tests for kie-camel itself and OSGI tests passed successfully with IBM Java.

**Comment by tarilabs** [comment]
@lucamolteni this confirmed be working also with IBM JDK both locally and on QE Jenkins. Good with you too?

**Comment by lucamolteni** [comment]
> @lucamolteni this confirmed be working also with IBM JDK both locally and on QE Jenkins. Good with you too?

Perfect thank you

**Review by mariofusco — APPROVED** [review]
